src/main/java/Storage.java

```diff
@@ -0,0 +1,40 @@
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+
```

Comment:
Same as TaskList class. Add Javadoc comments so that it will be easier to understand what this particular class is all about and what does the methods in the class do.